File: 2015/gara1/distributori/att/distributori.cpp
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

int rifornisci(int N, int M, int K, int D[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int D[MAXN];

int main() {
    FILE *fr, *fw;
    int N, M, K, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(3 == fscanf(fr, "%d %d %d", &N, &M, &K));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &D[i]));

    fprintf(fw, "%d\n", rifornisci(N, M, K, D));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed1/gara1/distributori/att/distributori.pas
function rifornisci(N, M, K: longint; var D: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    rifornisci := 42;
end;


const
    MAXN = 100000;
var
    N, M, K, i : longint;
    D          : array[0..MAXN-1] of longint;
    fr, fw     : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N, M, K);
    for i:=0 to N-1 do
        read(fr, D[i]);

    writeln(fw, rifornisci(N, M, K, D));
    close(fr);
    close(fw);
end.
